examples/jsm/postprocessing/AfterimagePass.js

```diff
@@ -89,6 +90,28 @@ class AfterimagePass extends Pass {
 
 	}
 
+	/**
+	 * Get the damping intensity. A higher value means a stronger after image effect.
+	 * 
+	 * @type {number}
+	 */
+	get damp() {
+
+		return this.uniforms[ 'damp' ].value;
+
+	}
+
+	/**
+	 * Set the damping intensity. A higher value means a stronger after image effect.
+	 * 
+	 * @type {number}
+	 */
```

Comment:
It's sufficient to document the getter. The docs for the setter can be removed.

Reply:
Removed